yarn.lock

```diff
@@ -17655,7 +17655,7 @@ inquirer-autocomplete-prompt@^1.4.0:
     run-async "^2.4.0"
     rxjs "^6.6.2"
 
-inquirer@8.2.2:
+inquirer@8.2.2, inquirer@^8.2.0:
```

Comment:
I'm not sure why this needed to be updated but I regenerated it because the yarn.lock check in CI wouldn't pass it.